src/pages/Profile/index.jsx

```diff
@@ -0,0 +1,215 @@
+              />
+              {errors.fullName && (
+                <p className={styles.card__error}>{errors.fullName.message}</p>
+              )}
+            </div>
+            <div>
+              <label htmlFor="email">E-mail</label>
+              <input
+                {...register("email")}
+                type="email"
+                name="email"
+                id="email"
+                required={errors.email ? true : false}
+              />
+              {errors.email && (
+                <p className={styles.card__error}>{errors.email.message}</p>
+              )}
+            </div>
+            <div>
+              <label htmlFor="birthdate">Data de nascimento</label>
+              <input
+                {...register("birthdate")}
+                type="date"
+                name="birthdate"
+                id="birthdate"
+                required={errors.birthdate ? true : false}
+              />
+              {errors.birthdate && (
+                <p className={styles.card__error}>{errors.birthdate.message}</p>
+              )}
+            </div>
+            <div>
+              <label htmlFor="profile">Perfil</label>
+              <input
+                {...register("profile")}
+                type="text"
+                name="profile"
+                id="profile"
+                required={errors.profile ? true : false}
+              />
+              {errors.profile && (
+                <p className={styles.card__error}>{errors.profile.message}</p>
+              )}
+            </div>
+            <div>
+              <InputDropdown
+                label="Gênero"
+                registerName="gender"
+                register={register}
+                name="gender"
+                id="gender"
+                data={[
+                  { label: "Feminino", value: "feminino" },
+                  { label: "Masculino", value: "masculino" },
+                  { label: "Transexual", value: "transexual" },
+                  { label: "Não-binário", value: "nao-binário" },
+                ]}
+              >
+              </InputDropdown>
+            </div>
+            <div>
+              <label htmlFor="organization">Organização</label>
+              <input
+                {...register("organization")}
+                type="text"
+                name="organization"
+                id="organization"
+                required={errors.organization ? true : false}
+              />
+              {errors.organization && (
+                <p className={styles.card__error}>{errors.organization.message}</p>
+              )}
+            </div>
+            <button className={styles.Profile__Button}
+              type="submit"
+              disabled={!formState.isDirty || !formState.isValid || formState.isSubmitSuccessful}>
+                Salvar dados
+            </button>
+          </form>
+          <Modal.Root isOpen={modalVisibility}>

```

Comment:
E pra ser assim? branco?
o alerta deve fica fora dá div, que você criará quando for ajusta background do layout  

![image](https://github.com/DAUUX/usarp-tool/assets/76105702/ca02347c-9ea7-4aea-ac30-94d62d829096)

Reply:
As mudanças feitas no modal estão no outro PR